Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_RipperHound.xml

```diff
@@ -26,19 +26,19 @@
 				</value>
 			</li>
 				
-					<li Class="PatchOperationReplace">
-		<xpath>/Defs/ThingDef[defName="AA_RipperHound"]/statBases/ArmorRating_Blunt</xpath>
-		<value>
-			<ArmorRating_Blunt>0.1</ArmorRating_Blunt>
-		</value>
-	</li>
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="AA_RipperHound"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>3</ArmorRating_Blunt>

```

Comment:
Leave it. It's a squishy wolf-thing on steroids--lucky to have any armor at all.